pkg/util/util.go

```diff
@@ -86,6 +91,41 @@ func StartSPDKTgtDaemon() error {
 	return nil
 }
 
+func StopSPDKTgtDaemon(timeout time.Duration) error {
+	processes, err := proc.FindProcessByCmdline("spdk_tgt")
+	if err != nil {
+		return errors.Wrap(err, "failed to find spdk_tgt")
+	}
+
+	var errs error
+	for _, process := range processes {
+		if err := process.Signal(syscall.SIGTERM); err != nil {
+			multierr.Append(errs, errors.Wrapf(err, "failed to send SIGTERM to spdk_tgt %v", process.Pid))
+		} else {
+			done := make(chan error, 1)
+			go func() {
+				_, err := process.Wait()
+				done <- err
+				close(done)
```

Comment:
nit: Saw the discussion before about if you need to close the channel. In this case, it's not needed, because it's scoped, and you already sent the err to the channel.